pages/settings/index.js

```diff
@@ -252,6 +253,20 @@ export default function Settings ({ ssrData }) {
             required
             append={<InputGroup.Text className='text-monospace'>sats</InputGroup.Text>}
           />
+          <Checkbox
+            label={
+              <div className='d-flex align-items-center'>disable freebies
+                <Info>
+                  <p>Some posts and comments can be created without paying. However, that content has limited visibility.</p>
+
+                  <p>If you disable freebies, you will always be prompted to pay for your posts and comments.</p>
+
+                  <p>If you attach a sending wallet, we disable freebies for you unless you have unchecked this box.</p>
```

Comment:
Mhh, the box is already unchecked by default (`disableFreebies = NULL`). So wouldn't "interacted" make more sense?

> If you attach a sending wallet, we disable freebies for you unless you interacted with this box and saved.